### ISSUE TITLE
review & approve monthly summaries

### ISSUE THREAD

**Issue by ramyasghub** [SBTCN/tanga2#4]

**Comment by ramyasghub** [comment]
@generalseven Do I have to get the past month summaries from all team members ? Kindly clarify.

**Comment by generalseven** [comment]
Yes, between the last day of the month and the 9th day of the next month.

- [ ] Shekhar - we need detailled specifications #7
- [ ] @ramyasghub same
- [ ] Brijendra + Shagun (2 hires per month + KPIs to be found on the last page of their contracts)

- [ ] Dhruv - 1500 original + publishable words per day, uploaded to WP

- [ ]  Shoaib Devops - velocity + scrum estimates
- [ ] Bharat Developer - velocity + scrum estimates

**Comment by ramyasghub** [comment]
@generalseven Summary updates of team :
Shekhar - Summary pending 
HR Leads - Reviewed
Others - Planned

**Comment by generalseven** [comment]
> HR Leads - Reviewed

Please post details.

**Comment by generalseven** [comment]
I see no update already asked two weeks ago.
